core/src/main/java/de/danoeh/antennapod/core/storage/mapper/FeedMediaCursorMapper.java

```diff
@@ -14,6 +15,7 @@
     /**
      * Create a {@link FeedMedia} instance from a database row (cursor).
      */
+    @SuppressLint("Range")
```

Comment:
Linting on the command line works fine on my device, even without this. I currently can't test in IntelliJ because 2021.2 is not available for my distro yet.

Reply:
What distro do you use?

Reply:
The error returned is
```
..\AntennaPod\core\src\main\java\de\danoeh\antennapod\core\storage\mapper\FeedMediaCursorMapper.java:39: Error: Value must be ? 0 [Range]
          switch (cursor.getInt(cursor.getColumnIndex(PodDBAdapter.KEY_HAS_EMBEDDED_PICTURE))) {
```

It might be a false positive, but I'm not really sure.

Reply:
Does using `getColumnIndexOrThrow` make the warning go away?

> What distro do you use?

Arch Linux, which is usually pretty fast upgrading packages

Reply:
`getColumnIndexOrThrow` does indeed make the warning go away :)

Fixed!